governance/default.go

```diff
@@ -605,13 +617,6 @@ func getGovernanceCacheKey(num uint64) common.GovernanceCacheKey {
 	return common.GovernanceCacheKey(params.GovernanceCachePrefix + "_" + v)
 }
 
-func (g *Governance) addIdxCache(num uint64) {
```

Comment:
Why did you remove this function?

Reply:
@ehnuje As far as I can remember, there was an issue for adding a cache for test code after idxCacheLock was added. It looks like it could be better. So I revived addIdxCache function and make AddGovernanceCacheForTest to use `addGovernanceCache` while not accessing `addIdxCache` directly.